runtime_scan/pkg/scanner/scanner.go

```diff
@@ -202,8 +199,13 @@ func (s *Scanner) initScan() error {
 			continue
 		}
 
-		// TODO: (idanf) verify if we need to read pod image pull secrets or just mount it to the scanner job
-		secrets := k8sutils.GetPodImagePullSecrets(s.clientset, pod)
+		// Fetch all image pull secrets from the pod so that we can
+		// inform the scanner where to check for credentials to pull
+		// the image to scan.
+		imagePullSecretNames := []string{}
+		for _, ips := range pod.Spec.ImagePullSecrets {
```

Comment:
now that we are not trying to fetch the secrets, we don't know if secret is missing, we need to do the mount secret `optional`. we don't want that the scanner will failed to run because of that (and we do see from time to time that secret in image pull secret is missing)